Userland/Applications/PixelPaint/ResizeImageDialog.cpp

```diff
@@ -67,6 +69,14 @@ ResizeImageDialog::ResizeImageDialog(Gfx::IntSize const& suggested_size, GUI::Wi
         }
     };
 
+    rescale_image_checkbox->on_checked = [this](bool is_checked) {
+        if (is_checked) {
+            m_rescale_image = true;
+        } else {
+            m_rescale_image = false;
+        }
```

Comment:
This could be simpler:
```suggestion
        m_rescale_image = is_checked;
```